src/api/registrationApi.ts

```diff
@@ -18,6 +18,16 @@ export const updateRegistration = async (registration: Registration) =>
     data: registration,
   });
 
+export const unpublishRegistration = async (
+  registration: Registration,
```

Comment:
holder kanskje å ta inn bare `identifier` her, da all payload ligger i neste argument?
```suggestion
  identifier: string,
```